### ISSUE TITLE
nvd3.org not quite responsive

### ISSUE THREAD

**Issue by stoft** [novus/nvd3#1949]

**Comment by jeznag** [comment]
You mean the responsive design isn't perfect?
![image](https://cloud.githubusercontent.com/assets/1034890/24336552/ebb00a9a-12dc-11e7-8d10-63d7ae07a64a.png)

**Comment by stoft** [comment]
Correct. Sorry for the short bug report last time, can't remember the context I was in but I think there were issues with uploading pics on my phone at the time. Here's a similar result on an iPhone 6s. I understand if charts overflow, but considering that nvd3 targets the browser it would be nice if at least the text wrapped to the screen width.
![img_0115](https://cloud.githubusercontent.com/assets/5726993/24497543/120775b4-153c-11e7-9b5f-552f05a118e4.PNG)
